packages/filebrowser/src/model.ts

```diff
@@ -104,7 +104,23 @@ export class FileBrowserModel implements IDisposable {
     };
     window.addEventListener('beforeunload', this._unloadEventListener);
     this._scheduleUpdate();
```

Comment:
Something should be done with `_scheduleUpdate` too - that should probably be going away with the conversion to using Poll.

Reply:
Looks like the `refresh()` method may need some work too.

Reply:
`_scheduleUpdate` has some logic around *minimum* refresh intervals, which I was endeavoring to leave intact for now. This is triggered upon `fileChanged` signals. I'm not aware of the history of that logic, but had thought that maybe there was some debouncing that was necessary.

If not, I'm happy to simplify this further, was just starting out with a more conservative approach.